packages/server/src/validators/eslint-validator.ts

```diff
@@ -0,0 +1,85 @@
+import { CLIEngine } from 'eslint';
+import chalk from 'chalk';
+import { Validator } from '../orchestrator/state';
+import { ValidationWarning, ValidationError } from '@bigtest/bundler';
+
+const EslintOptions: CLIEngine.Options = {
+  useEslintrc: false,
+  allowInlineConfig: true,
+  parserOptions: {
+    sourceType: 'module',
+    ecmaVersion: 9,
+    
+  },
+  baseConfig: {
+    root: true,
+    plugins: ['bigtest'],
+    rules: {
+      'bigtest/require-default-export': 'error',
+    },
+  },
+  extensions: ['.ts', '.tsx', '.js', '.jsx'],
+};
+
```

Comment:
What if we externalize this configuration into a `.babelrc` in the root of the `test/` directory. We can use our init script to deposit that boilerplate on installation. 

That way, users can tweak it to their hearts content, and really all we're doing is running eslint

Reply:
I think that was the case I was trying to avoid because this is only validating the things will stop bigtest working and not jumbled up with the user's wide range of formatting and whatever else they may have.

If we start inheriting rules from userland then it becomes more complicated.

Another way of doing this is to use [acorn](https://github.com/acornjs/acorn) or [estree-walker](https://github.com/Rich-Harris/estree-walker) which are lightweight parsers.

But I thought it would be good to kill the eslint rules and the file format check at the same time.

I think you can do the max depth rule with an AST also either eslint or one of the parsers above.